plugin/src/main/resources/config.yml

```diff
@@ -162,6 +162,11 @@ Tags:
   Block tags:
     # How many blocks can be read, max, before stopping the tag in place
     Max blocks: 1000000
+  # Whether tags should be allowed to automatically load currently unloaded chunks when attempting to access information.
+  # For example, when set to false, tags such as <chunk[10,10].entities> and <location[999999,70,999999,world].inventory> will error if the chunk is currently unloaded.
+  # If you are extremely worried about performance, it is recommended you disable this setting and modify your scripts appropriately, manually deciding when it is

```

Comment:
The change was not in the slightest related to performance reasons. I advise you seek to understand prior to changing.

Reply:
As you've chosen to exclude non-Discord community members from receiving valuable information, we can only assume. Feel free to open an issue with more information, if you'd like.

Reply:
@Morphan1 You are in no way excluded from accessing the public Denizen Discord at https://discord.gg/Q6pZGSR

Reply:
` non-Discord community members `
Aside from one of our members being blatantly excluded from accessing your server... 
I'm aware and have chosen not to participate in that medium.
You run a GitHub project. If you care about your programming community, try to keep information on GitHub. Once again, feel free to open an issue with more information. We'd love to have a proper note on this configuration option.